docs/reference/docs/reindex.asciidoc

```diff
@@ -522,9 +522,20 @@ reindex timed out.
 (integer) The number of documents that were successfully processed.
 
 `updated`::
-
++
+--
 (integer) The number of documents that were successfully updated.
```

Comment:
I would prefer to just clarify this line a bit and avoid the additional paragraphs.

```suggestion
(integer) The number of documents that were successfully updated, i.e., a document with same id already existed prior to reindex updating it.
```